rest-api/tools/import_organizations.py

```diff
@@ -217,6 +222,13 @@ def _populate_lat_lng_and_time_zone(self, site, existing_site):
         site.longitude = longitude
         if latitude and longitude:
           site.timeZoneId = self._get_time_zone(latitude, longitude)
+    else:
+      if site.googleGroup not in self.status_exception_list:
+        if site.siteStatus == self.ACTIVE:
+          if len(self.errors) == 0:
+            self.errors.append('Active sites must have valid addresses')
+            self.errors.append('The following sites were not geocoded')
```

Comment:
I'd add a single error message per site. This seems to be adding three errors here?

Reply:
it only did it the first iteration, but I changed the message anyway.